sdk/lib/crt/stdlib/gcvt.c

```diff
@@ -1,32 +1,73 @@
-/* Copyright (C) 1998 DJ Delorie, see COPYING.DJ for details */
+/*
+ * msvcrt.dll math functions
+ *
+ * Copyright 2003 Alexandre Julliard <julliard@winehq.org>
+ * Copyright 2010 Piotr Caban <piotr@codeweavers.com>
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
+ *
```

Comment:
```suggestion
```
suggested unless you synced majority of that file with Wine here (which was not stated in the PRs description).